packages/engine-core/src/library/LibraryEngine.ts

```diff
@@ -100,6 +101,29 @@ export class LibraryEngine implements Engine {
     initHooks(this)
   }
 
+  async transaction(action: 'start', options?: Tx.Options): Promise<Tx.Info>
+  async transaction(action: 'commit', info: Tx.Info): Promise<undefined>
+  async transaction(action: 'rollback', info: Tx.Info): Promise<undefined>
```

Comment:
void would make more sense from an API point of view?

Reply:
I see comment closed with 👍🏼 but type is still the same 🤔 
Did you want to change it?